src/components/Reviews.jsx

```diff
@@ -8,7 +8,7 @@ const Reviews = () => {
 
   useEffect(() => {
     setIsLoading(true);
-    API.get("/reviews")
+    Api.get("/reviews")
       .then((response) => {
         setList(response.data);
         setIsLoading(false);
```

Comment:
the ReviewCard is more like a ReviewList. You would expect a card to be an individual reviews data rather than the entire array